### ISSUE TITLE
Dmg calculator Banshee 13 bug.

### ISSUE THREAD

**Issue by Hydrange1324** [maphe/maphe.github.io#118]
Hi, I've tested my dmg against the theoretical dmg on B13, and I can conclude that the dmg calculator is doing something weird. Either that or the game is bugged, but I wanted to write to you firstly because I believe SG have their shit in the clear if I am the only one noticing this. 
So in short, the dmg I deal is less than the dmg calculator by a fair bit. I was missing runes to make my B13 3-man one shot, so I calculated the dmg needed to get the actual one shot on my Rem. When I finally got the runes (not for her but for Shadow Rose), I tried out my team and I ended up not killing the boss due to some form of dmg missing. I initially thought it could be Rem herself, as she is a new unit, but I asked in my guilds discord server for advice, and we played with the dmg calculator and concluded:
The B13 boss herself takes less dmg than we first thought. This is either a mistake from SG or a hidden dmg mitigation from her passive that states that she takes reduced dmg once she reaches phase 2. Some one said that they heard the boss takes about 8% less dmg when phase 2 triggers and onward, so we fidgeted with the dmg calculator and put in a dmg reduction of 8.44% and the exact number that shows up in my runs matched.
I have not confirmed with other units yet and I am going on vacation and don't have time to test further. 
Anyway, I just wanted to give a heads up to you, if indeed it is intended from SG that the boss takes about 8% less dmg than the calculator predicts.
Kind regards Hydrange

**Comment by Ningaal** [comment]
Hi,

I have been farming Banshee for this buff event. I built Vivian for one-shot. I also noticed Banshee and the first wave take less damage from Vivian. So I came here to report the issue and saw your post and the number you suggested (8.44%). Turns out I did not include **Atk% imprint** from my **leo.** Guess how much Leo's imprint percentage is? **8.4%!**
I added that number to the calculator and I got the same numbers the calculator predicted.

TLDR,
If you have Leo, make sure you include his imprint in the calc.
